start.html

```diff
@@ -70,6 +70,26 @@
 </table>
 
 
+<h2>Other third party drivers and plugins</h2>
+
+<p>Selenium can be extended through the use of plugins. Here are a number of plugins created and maintained by third parties. For more information on how to create your own plugin or have it listed, consult the docs.
+
+<p>Please note that these plugins are not supported, maintained, hosted, or endorsed by the Selenium project. In addition, be advised that the plugins listed below are not necessarily licensed under the Apache License v.2.0. Some of the plugins are available under another free and open source software license; others are only available under a proprietary license. Any questions about plugins and their license of distribution need to be raised with their respective developer(s).
+
+<table>
+ <tr>
+ <th>Browser</th><th>Latest version</th><th></th><th></th><th></th></tr>
+ <tr>
+  <td><a href="//sites.google.com/a/chromium.org/chromedriver/">Google Chrome driver</a></td>
```

Comment:
Technically chromedriver isn’t a plugin but a driver implementation, and the Selenium project are aiming for _all_ drivers to be supplied as external dependencies. But this isn’t a big matter.